src/components/CircularOutline/CircularOutline.js

```diff
@@ -95,6 +92,14 @@ class RoundedOutline extends Component<Props, State> {
     if (prevState.pathLength === 0 && this.state.pathLength !== 0) {
       this.setState({ finishedAllMountingSteps: true });
     }
+
+    if (this.state.pathLength !== null) {
+      const newPathLength = this.shapeNode.getTotalLength();
+
+      if (this.state.pathLength !== newPathLength) {
+        this.setState({ pathLength: newPathLength });
```

Comment:
I think within this conditional, we can also get the new `width` and `height`:

```js
if (this.state.pathLength !== newPathLength) {
  const {width, height} = this.wrapperNode.getBoundingClientRect();

  this.setState({ pathLength: newPathLength, width, height });
}
```

This way, we can still use `width` and `height` below.